DbSetup-kotlin/src/main/kotlin/com/ninja_squad/dbsetup_kotlin/DbSetupBuilder.kt

```diff
@@ -65,7 +65,7 @@ class DbSetupBuilder {
      * }
      * ```
      */
-    fun insertInto(table: String, configure: Insert.Builder.() -> Unit) {
+    inline fun insertInto(table: String, configure: Insert.Builder.() -> Unit) {
```

Comment:
Why just this one and not the others? Only because this is the more frequently used, or is there another reason I'm missing?